DockerTests.test010Install failing due to incorrect file permission assertion

<!--
Please fill out the following information, and ensure you have attempted
to reproduce locally
-->

**Build scan**:
https://gradle-enterprise.elastic.co/s/5iue6ecykwawg/tests/:qa:os:destructiveDistroTest.default-docker/org.elasticsearch.packaging.test.DockerTests/test010Install#1

**Repro line**:
Seems packaging tests don't produce repro lines.

**Reproduces locally?**:
Yes, it did [reproduce locally for me](https://gradle-enterprise.elastic.co/s/5vszfsstjqtcy).

**Applicable branches**:
Looks to be specific to `7.x`.

**Failure history**:
https://gradle-enterprise.elastic.co/scans/tests?search.buildToolTypes=gradle&search.buildToolTypes=maven&search.relativeStartTime=P7D&search.timeZoneId=America/Los_Angeles&tests.container=org.elasticsearch.packaging.test.DockerTests&tests.disabledDistributions=WyJvdXRjb21lOnNraXBwZWQiXQ&tests.sortField=FAILED&tests.test=test010Install&tests.unstableOnly=true

**Failure excerpt**:
```
org.elasticsearch.packaging.test.DockerTests > test010Install FAILED
    java.lang.AssertionError: File group of /usr/share/elasticsearch/config/elasticsearch.keystore is wrong
    Expected: "root"
         but: was "elasticsearch"
        at __randomizedtesting.SeedInfo.seed([DEB273B486698D2:29B487A82ACBC78A]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.elasticsearch.packaging.util.Docker.assertPermissionsAndOwnership(Docker.java:393)
        at org.elasticsearch.packaging.util.Docker.lambda$verifyOssInstallation$5(Docker.java:436)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
        at org.elasticsearch.packaging.util.Docker.verifyOssInstallation(Docker.java:436)
        at org.elasticsearch.packaging.util.Docker.verifyContainerInstallation(Docker.java:417)
        at org.elasticsearch.packaging.test.DockerTests.test010Install(DockerTests.java:103)

```

--- a/distribution/docker/src/docker/bin/docker-entrypoint.sh
+++ b/distribution/docker/src/docker/bin/docker-entrypoint.sh
@@ -7,7 +7,7 @@ umask 0002
 run_as_other_user_if_needed() {
   if [[ "$(id -u)" == "0" ]]; then
     # If running as root, drop to specified UID and run command
-    exec chroot --userspec=1000 / "${@}"
+    exec chroot --userspec=1000:0 / "${@}"
   else
     # Either we are running in Openshift with random uid and are a member of the root group
     # or with a custom --user
@@ -30,8 +30,8 @@ if [[ "$1" != "eswrapper" ]]; then
     # Without this, user could specify `elasticsearch -E x.y=z` but
     # `bin/elasticsearch -E x.y=z` would not work.
     set -- "elasticsearch" "${@:2}"
-    # Use chroot to switch to UID 1000
-    exec chroot --userspec=1000 / "$@"
+    # Use chroot to switch to UID 1000 / GID 0
+    exec chroot --userspec=1000:0 / "$@"
   else
     # User probably wants to run something else, like /bin/bash, with another uid forced (Openshift?)
     exec "$@"

--- a/qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java
+++ b/qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java
@@ -99,7 +99,6 @@ public class DockerTests extends PackagingTestCase {
     /**
      * Checks that the Docker image can be run, and that it passes various checks.
      */
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/60853")
     public void test010Install() {
         verifyContainerInstallation(installation, distribution());
     }

